common/model/bundle.c

```diff
@@ -87,6 +141,82 @@ void bundle_finalize(bundle_transactions_t *bundle, Kerl *const kerl) {
   }
 }
 
+retcode_t bundle_validator(bundle_transactions_t *const bundle,
+                           bundle_status_t *const status) {
+  retcode_t res = RC_OK;
+  iota_transaction_t *curr_tx = NULL;
+  int64_t index = 0, last_index = 0;
+  int64_t bundle_value = 0;
+  flex_trit_t bundle_hash[FLEX_TRIT_SIZE_243];
+  bool valid_sig = true;
+  Kerl kerl = {};
+
+  *status = BUNDLE_VALID;
```

Comment:
just in case of a bug, we should default to marking the bundle as invalid?

Reply:
I think should remove this, the default should be inited by the caller.